cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingPage {
+    private termsOfServiceCheckbox: string;
+
+    private checkoutBtn: string;
+
+    constructor() {
+      this.termsOfServiceCheckbox = "[id=cgv]";
+      this.checkoutBtn = "[name=processCarrier]";
+    }
+
+    public acceptTermsOfService(): void {
+      cy.get(this.termsOfServiceCheckbox).click();
+    }
+
+    public checkoutBtnEvent(): void {
```

Comment:
the methods on page objects generally should be called with the name of business actions, without referring for example to the element or events like in this case BtnEvent... instead of that, you can call it like proceedToCheckout, proceedToPayment or continueToPayment